app/tests/cases/system/student_simple.test.php

```diff
@@ -192,8 +193,8 @@ function($session) {
         $this->assertEqual($title, 'MECH 328 - Mechanical Engineering Design Project > Simple Evaluation > Results');
 
         //auto-release results message
-        $msg = $this->session->elementsWithWait(PHPWebDriver_WebDriverBy::ID, 'autoRelease_msg');
-        $this->assertTrue(!empty($msg));
+        // $msg = $this->session->elementsWithWait(PHPWebDriver_WebDriverBy::ID, 'autoRelease_msg');
```

Comment:
Any reason these tests are commented out?

Reply:
There is no element `autoRelease_msg` displyed on screen.  Seems the element only appears in test cases and CSS.  Hence they are commented out.
```
localhost:~/work/iPeer> grep -lr "autoRelease_msg" *
app/tests/cases/system/student_mixeval.test.php
app/tests/cases/system/student_simple.test.php
app/tests/cases/system/student_rubric.test.php
app/webroot/css/ipeer.css
```